src/components/structures/MatrixChat.js

```diff
@@ -589,7 +589,16 @@ module.exports = React.createClass({
                     var theAlias = MatrixTools.getCanonicalAliasForRoom(room);
                     if (theAlias) presentedId = theAlias;
                 }
-                self.notifyNewScreen('room/'+presentedId);
+
+                if (typeof presentedId != 'undefined') {
```

Comment:
Yeah, just saying you don't need the `typeof` here: `if (presentedId != 'undefined') {` would be the same.